checkstyle-suppressions.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0"?>
+
+<!DOCTYPE suppressions PUBLIC
+    "-//Puppy Crawl//DTD Suppressions 1.1//EN"
+    "http://checkstyle.sourceforge.net/dtds/suppressions_1_1.dtd">
+
+<suppressions>
+  <suppress checks="indentation"  files="ProcessModuleHandle.java"/>
```

Comment:
Please do not suppress the indentation check for the complete file. Suppress only the 3 methods where this is needed, for example using @SuppressWarnings("indentation") after enabling <module name="SuppressWarningsHolder" /> and <module name="SuppressWarningsFilter" />
as explained on https://checkstyle.sourceforge.io/config_filters.html#SuppressWarningsFilter
The rule should still catch any indentation issues we may add in other places of the file in future.

Reply:
Done